Makefile

```diff
@@ -124,6 +124,7 @@ build/gen-device-svd: ./tools/gen-device-svd/*.go
 
 gen-device-esp: build/gen-device-svd
 	./build/gen-device-svd -source=https://github.com/posborne/cmsis-svd/tree/master/data/Espressif-Community -interrupts=software lib/cmsis-svd/data/Espressif-Community/ src/device/esp/
+	./build/gen-device-svd -source=https://github.com/posborne/cmsis-svd/tree/master/data/Espressif -interrupts=software lib/cmsis-svd/data/Espressif/ src/device/esp/
```

Comment:
the repository https://github.com/posborne/cmsis-svd/tree/master/data/Espressif not exists and  `make gen-device-esp` failing:
```$ make gen-device-esp 
./build/gen-device-svd -source=https://github.com/posborne/cmsis-svd/tree/master/data/Espressif-Community -interrupts=software lib/cmsis-svd/data/Espressif-Community/ src/device/esp/
lib/cmsis-svd/data/Espressif-Community/esp32.svd
lib/cmsis-svd/data/Espressif-Community/esp8266.svd
./build/gen-device-svd -source=https://github.com/posborne/cmsis-svd/tree/master/data/Espressif -interrupts=software lib/cmsis-svd/data/Espressif/ src/device/esp/
cannot find input directory: lib/cmsis-svd/data/Espressif/
make: *** [gen-device-esp] Error 1
```

Reply:
If you're testing this locally, you need to run `git submodule update --init`.